defaults/main.yml

```diff
@@ -46,6 +46,15 @@ etckeeper_lowlevel_package_manager: |
 #
 etckeeper_vcs: 'git'
 
+# .. envvar:: etckeeper_vcs_username
+#
+# username to use when making VCS commits
```

Comment:
Another small thing. Can you change the style to match the rest of the file. E.g. start a sentence with a upper case letter and end with a full stop? Sorry for the many comments :wink: